app/serializers/api/admin/line_item_serializer.rb

```diff
@@ -13,7 +13,7 @@ def units_variant
     Api::Admin::UnitsVariantSerializer.new(object.variant).serializable_hash
   end
 
-  def unit_value
-    object.unit_value.to_f
+  def final_weight_volume
+    object.final_weight_volume.to_f
```

Comment:
Actually, unit_value can be `nil` and `nil.to_f` is `0.0`. Maybe that was the idea behind.